2-mandatory/6-budgets.js

```diff
@@ -11,301 +11,316 @@ Should give return the answer of 62600.
-      age: 75,
-      budget: 9077,
-    },
-    {
-      name: "Caroline",
-      age: 33,
-      budget: 8301,
-    },
-    {
-      name: "Grant",
-      age: 49,
-      budget: 9942,
-    },
-    {
-      name: "Tammi",
-      age: 58,
-      budget: 7447,
-    },
-    {
-      name: "Mayra",
-      age: 73,
-      budget: 3500,
-    },
-    {
-      name: "Goodwin",
-      age: 28,
-      budget: 4223,
-    },
-    {
-      name: "Cathryn",
-      age: 57,
-      budget: 2517,
-    },
-    {
-      name: "Stuart",
-      age: 20,
-      budget: 8208,
-    },
-    {
-      name: "Lina",
-      age: 45,
-      budget: 5131,
-    },
-    {
-      name: "Hampton",
-      age: 62,
-      budget: 3423,
-    },
-    {
-      name: "Coleen",
-      age: 49,
-      budget: 5377,
-    },
-    {
-      name: "Jamie",
-      age: 56,
-      budget: 8361,
-    },
-    {
-      name: "Brigitte",
-      age: 77,
-      budget: 1460,
-    },
-    {
-      name: "Colette",
-      age: 41,
-      budget: 3419,
-    },
-    {
-      name: "Susan",
-      age: 45,
-      budget: 8097,
-    },
-    {
-      name: "Shaffer",
-      age: 75,
-      budget: 3883,
-    },
-    {
-      name: "Clements",
-      age: 74,
-      budget: 3141,
-    },
-    {
-      name: "Cobb",
-      age: 58,
-      budget: 6058,
-    },
-    {
-      name: "Ochoa",
-      age: 25,
-      budget: 7916,
-    },
-    {
-      name: "Anita",
-      age: 72,
-      budget: 2678,
-    },
-    {
-      name: "Carolyn",
-      age: 51,
-      budget: 2781,
-    },
-    {
-      name: "Beard",
-      age: 72,
-      budget: 9433,
-    },
-    {
-      name: "Bender",
-      age: 26,
-      budget: 8824,
-    },
-    {
-      name: "Rich",
-      age: 37,
-      budget: 3312,
-    },
-    {
-      name: "Angelina",
-      age: 48,
-      budget: 8997,
-    },
-    {
-      name: "Cecelia",
-      age: 30,
-      budget: 5044,
-    },
-    {
-      name: "Nixon",
-      age: 59,
-      budget: 5178,
-    },
-    {
-      name: "Alana",
-      age: 30,
-      budget: 7976,
-    },
-    {
-      name: "Mcconnell",
-      age: 54,
-      budget: 2419,
-    },
-    {
-      name: "Cunningham",
-      age: 21,
-      budget: 5659,
-    },
-    {
-      name: "Concetta",
-      age: 58,
-      budget: 8269,
-    },
-    {
-      name: "Natalie",
-      age: 27,
-      budget: 3078,
-    },
-    {
-      name: "Lee",
-      age: 46,
-      budget: 3743,
-    },
-    {
-      name: "Monica",
-      age: 18,
-      budget: 8701,
-    },
-    {
-      name: "Lauri",
-      age: 61,
-      budget: 7849,
-    },
-  ])).toEqual(289531);
+  expect(
+    getBudgets([
+      {
+        name: "Reba",
+        age: 73,
+        budget: 9657,
+      },
+      {
+        name: "Tessa",
+        age: 77,
+        budget: 2585,
+      },
+      {
+        name: "Margery",
+        age: 48,
+        budget: 2269,
+      },
+      {
+        name: "Manuela",
+        age: 75,
+        budget: 4761,
+      },
+      {
```

Comment:
At the risk of repeating myself...
Imagine you ask a colleague to review a PR. If they see this page they'll assume that you made a lot of changes and it'll take them a lot of time. And then when reviewing they'll see a lot of unnecessary changes like this one, and that will frustrate them. 
Once more: each commit should do one thing only: answer the exercise's question. Anything else (reformatting, typos, etc) goes in another commit.